### DISCUSSION TITLE
vc-drawings中绘制的pin可以针对每一个配置文字text吗

### DISCUSSION THREAD

**Discussion by 1510171848** [zouyaoji/vue-cesium#495] [Q&A]
![Snipaste_2023-12-27_17-59-59](https://github.com/zouyaoji/vue-cesium/assets/43635418/c6618f9c-80e8-47f5-afd0-d663aaca7e74)

**Comment by zouyaoji** [comment]
参考一下 https://github.com/zouyaoji/vue-cesium/discussions/206

拿到renderDatas去改
